Vagrantfile

```diff
@@ -0,0 +1,54 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile made from
+# https://github.com/brd/packer-freebsd
+# https://github.com/wunki/vagrant-freebsd
+
+require 'yaml'
+
+Vagrant.configure("2") do |config|
+  if Vagrant.has_plugin?("vagrant-proxyconf")
+      if ENV.has_key?('http_proxy')
+          config.proxy.http     = ENV['http_proxy']
+      end
+      if ENV.has_key?('https_proxy')
+          config.proxy.https    = ENV['https_proxy']
+      end
+      if ENV.has_key?('no_proxy')
+          config.proxy.no_proxy = ENV['no_proxy']
+      end
+  end
+  config.vm.guest = :freebsd
+  config.vm.box = 'freebsd/FreeBSD-10.3-STABLE'
+  config.ssh.shell = "sh"
+  config.vm.synced_folder ".", "/vagrant", disabled: true
+  config.vm.base_mac = "080027D14C66"
```

Comment:
Does the MAC need to be specified? I seem to recall from my one foray into Vagrant that it could be done automatically but it may have been the plugin (we used libvirt) that did it for me.